wiki_docs/Create How_GTM_works

```diff
@@ -0,0 +1,47 @@
+User can add or manage the AdWords, Google analytics, Flood-light and non-Google tags using the GTM.
```

Comment:
How about: **Google Tag Manager (GTM) ** is a tool that allows you to to add tracking tags to your website or app. Contributors, such as marketers, can help track data on your site without having to add any hard code to your code base. 

A **tracking tag** enables you to collect data about your users, such as how many times they view a website page or how many times they click on a button. GTM connects with third party services like Google Analytics, Google AdWords, etc.